### PR TITLE
[11.0] migrate website_menu_by_user_status

### PR DESCRIPTION
**Pull request by ddufresne**

### CODE REVIEW

File: website_menu_by_user_status/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+
+Contributors
+------------
+* Bruno Joliveau <bruno.joliveau@savoirfairelinux.com>
+* Jordi Riera <jordi.riera@savoirfairelinux.com>
+* Meyomesse Gilles <meyomesse.gilles@gmail.com>
+* David Dufresne
+
+More information
```
Comment (simahawk) [inline website_menu_by_user_status/README.rst]: update README according to https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: website_menu_by_user_status/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013-2017 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Website Menu By User Display',
+    'version': '11.0.1.0.0',
+    'author': 'Savoir-faire Linux,Odoo Community Association (OCA)',
+    'website': 'http://www.savoirfairelinux.com',
```
Comment (simahawk) [inline website_menu_by_user_status/__manifest__.py]: https://github.com/OCA/website

File: website_menu_by_user_status/models/website_menu.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```
Comment (simahawk) [inline website_menu_by_user_status/models/website_menu.py]: remove coding headers from *.py

**Review by simahawk — CHANGES_REQUESTED** [review]

File: website_menu_by_user_status/i18n/website_menu_by_user_status.pot [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Translation of Odoo Server.
```
Comment (simahawk) [inline website_menu_by_user_status/i18n/website_menu_by_user_status.pot]: remove this file

**Comment by simahawk** [comment]
I'd squash some commits together here
image

**Comment by ddufresne** [comment]
@simahawk done

**Review by simahawk — APPROVED** [review]
LGTM

**Review by njeudy — APPROVED** [review]
code and tests

**Comment by flotho** [comment]
Hi,
Any chance to be backported in v10.0 ?

**Comment by ddufresne** [comment]
@flotho I don't intend on working on this module for the version 10.0. Be my guess if you wish to backport it. Just tell me if you need a reviewer.

**Pull request merged**